express-postgres/server.ts

```diff
@@ -0,0 +1,89 @@
+import express from 'express';
+import { ClientError, errorMiddleware } from './lib/index.js';
+import pg from 'pg';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+      select
+        "title",
+        "filmId"
+      from "films"
+      order by "replacementCost" desc;
+    `;
+    const result = await db.query(sql);
+    const films = result.rows;
+    if (!films) {
+      throw new ClientError(404, 'films not found');
+    }
```

Comment:
This wouldn't be a client error, this would be a server error. Client errors are when the client forgets to give us the content we need to make a request. If there are no films found, that's a server error. So we don't need to check for this, because our `catch` will catch the error and throw it in the middleware